R-package/R/xgb.plot.multi.trees.R

```diff
@@ -24,44 +20,45 @@
 #' Moreover, the trees tend to reuse the same features.
 #'
 #' The function projects each tree onto one, and keeps for each position the
-#' \code{features_keep} first features (based on the Gain per feature measure).
+#' `features_keep` first features (based on the Gain per feature measure).
 #'
 #' This function is inspired by this blog post:
-#' \url{https://wellecks.wordpress.com/2015/02/21/peering-into-the-black-box-visualizing-lambdamart/}
+#' <https://wellecks.wordpress.com/2015/02/21/peering-into-the-black-box-visualizing-lambdamart/>
 #'
-#' @return
-#'
-#' When \code{render = TRUE}:
-#' returns a rendered graph object which is an \code{htmlwidget} of class \code{grViz}.
-#' Similar to ggplot objects, it needs to be printed to see it when not running from command line.
-#'
-#' When \code{render = FALSE}:
-#' silently returns a graph object which is of DiagrammeR's class \code{dgr_graph}.
-#' This could be useful if one wants to modify some of the graph attributes
-#' before rendering the graph with \code{\link[DiagrammeR]{render_graph}}.
+#' @inherit xgb.plot.tree return
 #'
 #' @examples
 #'
-#' data(agaricus.train, package='xgboost')
+#' data(agaricus.train, package = "xgboost")
+#'
 #' ## Keep the number of threads to 2 for examples
 #' nthread <- 2
 #' data.table::setDTthreads(nthread)
 #'
 #' bst <- xgboost(
-#'   data = agaricus.train$data, label = agaricus.train$label, max_depth = 15,
-#'   eta = 1, nthread = nthread, nrounds = 30, objective = "binary:logistic",
-#'   min_child_weight = 50, verbose = 0
+#'   data = agaricus.train$data,
+#'   label = agaricus.train$label,
+#'   max_depth = 15,
+#'   eta = 1,
+#'   nthread = nthread,
+#'   nrounds = 30,
+#'   objective = "binary:logistic",
+#'   min_child_weight = 50,
+#'   verbose = 0
 #' )
 #'
 #' p <- xgb.plot.multi.trees(model = bst, features_keep = 3)
 #' print(p)
 #'
 #' \dontrun{
 #' # Below is an example of how to save this plot to a file.
-#' # Note that for `export_graph` to work, the DiagrammeRsvg and rsvg packages must also be installed.
+#' # Note that for export_graph() to work, the {DiagrammeRsvg} and {rsvg} packages
```

Comment:
Is the single quote redundant for `export_graph`?

Reply:
Since the sentence is behind a comment sign, it is not translated to Markdown/rd. This is why (a) no backticks and (b) curly braces around package names.

Reply:
got it. Thank you for the explanation.